src/main/java/crazypants/enderio/material/MaterialRecipes.java

```diff
@@ -109,7 +108,7 @@ public static void addRecipes() {
     } else {
       GameRegistry.addShapedRecipe(cbc, "ggg", "scs", "ggg", 'g', Blocks.gravel, 's', Blocks.sand, 'c', Items.clay_ball);
     }
-    FurnaceRecipes.smelting().func_151394_a(binderComposite, conduitBinder, 0);
+    GameRegistry.addSmelting(binderComposite, conduitBinder, (float) 0.5);
```

Comment:
Conforms to vanilla XP amounts, rather than have none entirely. Can be changed if wanted.